diagnostics-program/startDiag.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 #Run diag
-rm /opt/nebraDiagnostics/html/index.html
-rm /opt/nebraDiagnostics/html/initFile.txt
-#rm /var/data/public_keys
+rm -f /opt/nebraDiagnostics/html/index.html
+rm -f /opt/nebraDiagnostics/html/initFile.txt
+#rm -f /var/data/public_keys
 #cp /opt/nebraDiagnostics/html/index.html.template /opt/nebraDiagnostics/html/index.html
 
 nginx
 #sleep 30
-#rm /opt/nebraDiagnostics/html/index.html
+#rm -f /opt/nebraDiagnostics/html/index.html
```

Comment:
Also, please remove all commented out blocks. We can always retrieve them from git if needed.

Reply:
I can delete all commented out blocks, but they were there before me.

Reply:
Oh i know. It's just part of the clean up effort.

Reply:
I understand. Done.